devel/shfmt/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        mvdan sh 2.2.1 v
+name                shfmt
+categories          devel
+platforms           darwin
+maintainers         {@amake madlon-kay.com:aaron+macports} openmaintainer
+license             BSD
+
+description         Autoformat shell script source code
+
+long_description    A shell script formatter supporting POSIX shell, Bash, and mksh.
+
+depends_build       port:go
+use_configure       no
+
+checksums           rmd160  04c9a557c1595a7106d60c94f0df1323eddcc61f \
+                    sha256  efdf0aaf78357a19ad065155a9215c136eb571ca19b697f5f9f2662797eff4de
+
+post-extract {
+    xinstall -d ${worksrcpath}/src/mvdan.cc/
+    ln -sf ${worksrcpath} ${worksrcpath}/src/mvdan.cc/${github.project}
+}
+
+build.cmd           go
+build.target        build
+build.args          -a -tags 'production brew' mvdan.cc/sh/cmd/shfmt
+build.env           GOPATH=${worksrcpath} \
+                    CC=${configure.cc} \
+                    DYLD_INSERT_LIBRARIES=''
```

Comment:
This is only required because Go ports do not build in trace mode. Just remove this and don't build in trace mode. We will eventually provide a way to mark ports that do not build and trace mode to allow selectively disabling it.